jetty-servlet/src/main/java/org/eclipse/jetty/servlet/ServletHandler.java

```diff
@@ -452,17 +455,18 @@ public void doScope(String target, Request baseRequest, HttpServletRequest reque
                 String servletPath = pathSpec.getPathMatch(target);
                 String pathInfo = pathSpec.getPathInfo(target);
 
+                HttpServletMapping httpServletMapping = mapping.getHttpServletMapping(servletPath);
                 if (DispatcherType.INCLUDE.equals(type))
                 {
                     baseRequest.setAttribute(RequestDispatcher.INCLUDE_SERVLET_PATH, servletPath);
```

Comment:
It's not really necessary, but lets do the Attributes wrapping trick here as well!  Do we unset these attributes ever?  We should!